static/templates/settings/settings_checkbox.handlebars

```diff
@@ -1,7 +1,8 @@
-<div class="input-group">
+<div class="input-group {{#is_false unless}}control-label-disabled{{/is_false}}">
```

Comment:
`unless` sounds confusing here, can we rename it to `enabled` or something..?

Reply:
yeah... actually I also can't think of better name here that's why just put `unless` :), for `enable` sounds similar to `is_checked`. Maybe `is_label_disabled` or `is_checkbox_disabled`?

Reply:
`is_checkbox_enabled`/`is_label_enabled`/`is_control_enabled` sounds good, `is_checkbox_disabled` shows reverse flag from argument we pass.

Reply:
How about `is_parent_setting_enabled`